internal/configuration/command_update.go

```diff
@@ -0,0 +1,96 @@
+package configuration
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/form"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newUpdateCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:               "update [key] [value]",
```

Comment:
These are required, so we use `<>`. We should write a linter rule to catch this...
```suggestion
		Use:               "update <key> <value>",
```